setup.py

```diff
@@ -0,0 +1,76 @@
+import setuptools
+from setuptools.command.install import install
+from setuptools.command.test import test
+from setuptools import setup, Command
+import subprocess
+import os
+import unittest
+
+BASEPATH = os.path.dirname(os.path.abspath(__file__))
+NFSV4_PATH = os.path.join(BASEPATH, 'nfsv4')
+SFS_PATH = os.path.join(BASEPATH, 'sfs')
+
+class SfsInstall(install):
+    def run(self):
+        subprocess.call('make', cwd = NFSV4_PATH)
+        install.run(self)
+
+class SfsTest(test):
+    description = "run sfs_test.py using unittest"
+    user_options = [
+        ('ip=', None, 'IP of NFS4_SERVER.'),
+    ]
+    def initialize_options(self):
+        self.ip = None
+    
+    def finalize_options(self):
+        pass
+    
+    def run(self):
+        test_loader = unittest.TestLoader()
+        test_suite = test_loader.discover('test', pattern='sfs_test.py')
+        
+        import sfs
+        try:
+            sfs.mount(self.ip)
+        except Exception as e:
+            print("NFS SSERVER IP: ", self.ip)
```

Comment:
Extra S in server. Also, please enhance the error message to provide guidance on how to fix the problem.